align_system/utils/alignment_utils.py

```diff
@@ -53,7 +59,9 @@ def _select_min_dist_choice(self, choices, dists, misaligned=False, probabilisti
             # maximize over non-inverted distances
 
             # Convert distances to probabilities
-            probs = [dist/sum(dists) for dist in dists]
+            # Avoid divide by zero if all distances are 0
+            total_dist = sum(dists) + eps * len(dists)
```

Comment:
Does it make more sense here to do `sum(dists) * len(dists) + eps` (we don't need to multiply the epsilon right?)

Reply:
(Or maybe I'm just confused about the intent here)

Reply:
Yeah, I think we should remove `len(dists)` from the equation entirely `sum(dists) + eps`

Reply:
Alright what do we think now? (`probs = [dist/(sum(dists) + eps) for dist in dists]`)